cdxev/amend/operations.py

```diff
@@ -58,12 +123,19 @@ def _add_bom_ref(self, component: dict) -> None:
             component["bom-ref"] = str(uuid.uuid4())
 
 
-class CompositionsOperation(Operation):
+@default
+class Compositions(Operation):
     """
-    According to https://www.ntia.gov/files/ntia/publications/sbom_minimum_elements_report.pdf
-    "known unknowns" should be stated, as we can't guarantee completeness,
-    compositions should be marked as 'incomplete' for SBOMs.
-    This operation erases existing compositions and then adds all components as 'incomplete'.
+    Declares all component compositions as 'incomplete'.
```

Comment:
As you already removed to add `metadata.component` and state in your commit that it fixes #161, did you change your mind about the `unknown`?